enterprise/models.py

```diff
@@ -2235,14 +2237,27 @@ class Meta:
         'placeholder {course_name} will be replaced with the name of the course or program that was enrolled in.'
     )
 
+    DEFAULT_TEMPLATE_TYPE = 'DE'
+    BULK_ENROLL_TEMPLATE_TYPE = 'BE'
+    template_type_choices = [
+        (DEFAULT_TEMPLATE_TYPE, 'Default Enrollment Template'),
+        (BULK_ENROLL_TEMPLATE_TYPE, 'Bulk Enrollment Template'),
+    ]
+
     plaintext_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     html_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     subject_line = models.CharField(max_length=100, blank=True, help_text=SUBJECT_HELP_TEXT)
-    enterprise_customer = models.OneToOneField(
+    enterprise_customer = models.ForeignKey(
```

Comment:
in contrast to before, now we can have multiple templates per customer with type specified.

Reply:
This is actually not a required change. We just need to remove the requirement that it be present. (ie make it nullable)